storops/unity/resource/system.py

```diff
@@ -148,6 +148,25 @@ def get_ip_port(self, _id=None, name=None, **filters):
         return self._get_unity_rsc(UnityIpPortList, _id=_id, name=name,
                                    **filters)
 
+    @version(">=4.1")
+    def get_file_port(self):
+        """Returns ports list can be used by File
+
+        File ports includes ethernet ports and link aggregation ports.
+        """
+        eths = self.get_ethernet_port()
+        las = self.get_link_aggregation()
+        return eths + las
+
+    @version("<4.1")   # noqa

```

Comment:
This is used to make the pep8 happy to get rid of the duplicated definition error.